runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -797,6 +847,56 @@ public void onGraphicChanged(GraphicChanged event)
 	@Subscribe
 	public void onItemContainerChanged(ItemContainerChanged itemContainerChanged)
 	{
+		if (config.showAntiPoison() && this.prevItems != null)
+		{
+			Item[] prevInv = this.prevItems;
```

Comment:
```suggestion
			Item[] prevInv = prevItems;
```

Reply:
Also, `prevInv` is never mutated, so it would be fine to pass `prevItems` into `getPreviousItemId()`, then null it after the `itemID` checks. Copying it here doesn't seem to be necessary.